### PR TITLE
feat: add cognitiveServicesAccount service

### PR DESCRIPTION
**Pull request by hjaraujof**
Closes CG-1032

### CODE REVIEW

File: src/services/cognitiveServicesAccount/format.ts [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      privateEndpointConnections = [],
+      publicNetworkAccess,
+      apiProperties: {
+        qnaRuntimeEndpoint,
+        qnaAzureSearchEndpointKey,
+        qnaAzureSearchEndpointId,
+        statisticsEnabled,
+        eventHubConnectionString,
+        storageAccountConnectionString,
+        aadClientId,
```
Comment (tyler-dunkel) [inline src/services/cognitiveServicesAccount/format.ts:94]: any way to build connections to this ad data?

Comment (hjaraujof) [reply src/services/cognitiveServicesAccount/format.ts:94]: Yes, I believe it can be done. Let me trial some more 😄

Comment (hjaraujof) [reply src/services/cognitiveServicesAccount/format.ts:94]: I added the connection logic and field, but I'm unable to deploy the needed resources to test this in our sandbox account

Comment (tyler-dunkel) [reply src/services/cognitiveServicesAccount/format.ts:94]: ok sounds good. I suppose we can just 🤞 lol

**Review by tyler-dunkel — APPROVED** [review]

**Pull request merged**